pkg/minikube/node/node.go

```diff
@@ -78,8 +78,8 @@ func Add(cc *config.ClusterConfig, n config.Node, delOnFail bool) error {
 	return err
 }
 
-// Delete stops and deletes the given node from the given cluster
-func Delete(cc config.ClusterConfig, name string) (*config.Node, error) {
+// remove drains and deletes the given node in the given cluster.
+func remove(cc config.ClusterConfig, name string) (*config.Node, error) {
```

Comment:
in addition to draining a node first, we are also removing it from the cluster right after - should we name the func eg removeNodeFromCluster?

Reply:
I think drainNode is a better name. since it is called by the Delete func is okay, I just want more specific names, so I know this func does drain and this other one actually deletes. so lets go with drainNode

Reply:
sure, i've renamed it